config.yml

```diff
@@ -0,0 +1,246 @@
+---
+CI_HELPER_VERSION: &CI_HELPER_VERSION 0.0.6
+ci_helper_test: &ci_helper_test ci-helper version -e
+docker_hub_namespace: &docker_hub_namespace ekino/docker-buildbox
+
+
+ansible:
+  "2.5.5-r0":
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      PYTHON_VERSION: 2.7.15-r1
+      VERSION: 2.5.5-r0
+    cmd_test:
+      - ansible --version
+      - ansible-playbook --version
+      - python --version
+
+arachni:
+  "1.5":
+    build_args:
+      ARACHNI_VERSION: 1.5.1
+      ARACHNI_WEB_UI_VERSION: 0.5.12
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+    cmd_test:
+      - arachni --version
+      - *ci_helper_test
+
+aws:
+  "1":
+    cmd_test:
+      - aws --version
+      - pip list | grep boto3
+      - pip list | grep pyyaml
+      - pip --version
+      - pipenv --version
+      - *ci_helper_test
+    build_args:
+      CI_HELPER_VERSION: *CI_HELPER_VERSION
+      PIP_VERSION: 18.1
+      PIPENV_VERSION: 2018.10.13
+      PY_YAML_VERSION: 3.13
+
+chrome:
```

Comment:
Chrome installs CI Helper too so you need to add the test.

By the way I can see that you didn't inject `CI_HELPER_VERSION` but the build is working anyway : It might be a silent failure.

Reply:
And test modd also

Reply:
Aaaand you can also centralize MODD_VERSION like you did for CI Helper I think